core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -303,16 +315,59 @@ public final String getBuildNowText() {
     }
 
     /**
-     * Marker for job using this mixin.
+     * Marker for job using this mixin, and default implementations of many methods.
      */
-    public interface ParameterizedJob extends hudson.model.Queue.Task, hudson.model.Item {
+    public interface ParameterizedJob<JobT extends Job<JobT, RunT> & ParameterizedJobMixIn.ParameterizedJob<JobT, RunT> & Queue.Task, RunT extends Run<JobT, RunT> & Queue.Executable> extends BuildableItem {
```

Comment:
:bug: It may sound ridiculous, but the original interface didn't actually require this parameterized job to be buildable. Don't ask why anyone would need parameters then...

By extending `BuildableItem` you made these items Buildable, which may impact UI (there will be build button, option to run the job in CLI, etc.). It may cause regressions in Job types, which are not supposed to be buildable. It needs more investigation before applying